open_spiel/algorithms/is_mcts_test.cc

```diff
@@ -28,81 +28,81 @@ namespace {
 constexpr const int kSeed = 93879211;
 
 void PlayGame(const Game& game, algorithms::ISMCTSBot* bot, std::mt19937* rng) {
-  std::unique_ptr<State> state = game.NewInitialState();
-  while (!state->IsTerminal()) {
-    std::cout << "State:" << std::endl;
-    std::cout << state->ToString() << std::endl;
-
-    Action chosen_action = kInvalidAction;
-    if (state->IsChanceNode()) {
-      chosen_action =
-          SampleAction(state->ChanceOutcomes(), absl::Uniform(*rng, 0.0, 1.0))
-              .first;
-    } else {
-      chosen_action = bot->Step(*state);
+    std::unique_ptr<State> state = game.NewInitialState();
```

Comment:
Please revert changes that only do whitespace formatting.

(i.e. this entire file)